lang-model/pom.xml

```diff
@@ -11,12 +11,12 @@
 	<modelVersion>4.0.0</modelVersion>
 
 	<parent>
-		<groupId>jakarta.enterprise</groupId>
-		<artifactId>jakarta.enterprise.cdi-parent</artifactId>
+		<groupId>jakarta.cdi</groupId>
+		<artifactId>jakarta.cdi-parent</artifactId>
 		<version>5.0.0.Beta1-SNAPSHOT</version>
 	</parent>
 
-	<artifactId>jakarta.enterprise.lang-model</artifactId>
+	<artifactId>jakarta.lang-model</artifactId>
```

Comment:
This should IMHO be `jakarta.lang-model-api` or even better `jakarta.cdi-lang-model-api`.

Reply:
I was about to add a comment here myself - I was considering that but I know we originally wanted to offer this API outside of CDI so stripping of the `cdi` affix in the artifact ID made sense to me.
That said, we haven't had that usage so far so I am not feeling strongly about it :shrug: